packages/manager/modules/emailpro/src/domain/emailpro-domain.controller.js

```diff
@@ -147,6 +165,10 @@ export default /* @ngInject */ (
     }
   }
 
+  $scope.setDkimColorClass = function setDkimColorClass(status) {
+    return DKIM_STATUS_TO_CLASS_MAP[status] || '';
+  };
+
```

Comment:
```suggestion
  
```
This function is useless, you can use directly your constant object in your view (html file)